<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxesSettings</class>
 <widget class="QWidget" name="AxesSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>177</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X minvalue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>X maxvalue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Y minvalue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Y maxvalue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Z minvalue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Z maxvalue</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="xMin"/>
        </item>
        <item>
         <widget class="QLineEdit" name="xMax"/>
        </item>
        <item>
         <widget class="QLineEdit" name="yMin"/>
        </item>
        <item>
         <widget class="QLineEdit" name="yMax"/>
        </item>
        <item>
         <widget class="QLineEdit" name="zMin"/>
        </item>
        <item>
         <widget class="QLineEdit" name="zMax"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ID">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
